Configuration.TestHarness/ConsulRx.Configuration.TestHarness.csproj

```diff
@@ -11,4 +11,7 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="8.0.*" />
```

Comment:
I didn't explicitly add this.  Some googling tells me this is to generate better source links for nupkgs